kontroller/qml/kontroller.qml

```diff
@@ -127,7 +127,7 @@ ApplicationWindow {
         }
         onDownloadError: {
             var theFile = file;
-            var message = qsTr("Download of %1 failed with error %2").arg(theFile).arg(error);
+            var message = qsTr("Could not download %1 because %2").arg(theFile).arg(error);
```

Comment:
Not quite sure what %2 is, but I think this works.

Reply:
It should. %2 is a string describing the error.